bundle/compliance/cis_gcp/rules/cis_1_12/rule.rego

```diff
@@ -0,0 +1,20 @@
+package compliance.cis_gcp.rules.cis_1_12
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+default is_valid = false
+
+is_valid if {
+	not startswith(data_adapter.resource.data.name, "projects/")
```

Comment:
I'm not sure why are you're using this `is_valid` pattern, am I missing some rego thing here?

Reply:
to me a utility function is only valuable if it is utilized in multiple places. in this case, and many others, this is only used once, so i my personal preference is to keep everything in one file, instead of:
1 - audit function in one place
2 - rule in a different place
3 - a test that effectively tests the audit function 

just because we're doing something now, doesn't mean we have to keep doing it. if this was an existing benchmark, i'd probably go with being consistent. given it's a new one, and there are no other consumers for this logic, extracting it to a utility seems more like a burden than an advantage of some sort. in short - it's simpler.

Reply:
removed `is_valid` from project scope, added a descriptive variable name:
```
is_project_apikey := startswith(data_adapter.resource.data.name, "projects/")
```

to be used instead